apps/chat_server/lib/chat_server/contexts/starred_messages/create_starred_message.ex

```diff
@@ -0,0 +1,28 @@
+defmodule ChatServer.CreateStarredMessage do
+  require Logger
+
+  alias ChatServer.Schema
+
+  def call(message_id, user) do
+    with message <- Schema.Message.get(message_id),
+      {:ok, record} <- create_record(message, user) do
```

Comment:
Looks great as a `with` statement. Minor note on formatting, consecutive `with` statements should be lined up with the first. In other words, this line should be indented 5 instead of 2:

```elixir
with message <- Schema.Message.get(message_id),
     {:ok, record} <- create_record(message, user) do
  {:ok, Schema.StarredMessage.preload(record, [:message, :user])}
```